operate/schema/src/main/java/io/camunda/operate/schema/indices/AbstractIndexDescriptor.java

```diff
@@ -45,4 +46,17 @@ public String getSchemaClasspathFilename() {
       return String.format(SCHEMA_CREATE_INDEX_JSON_OPENSEARCH, getIndexName());
     }
   }
+
+  // FIXME clean this up when Operate exporter is converted to using Spring
+  public String getIndexPrefix() {
```

Comment:
Is it possible for more than one thread to call either of these methods on the same index descriptor? If so, we need to take that into account to avoid the edge case where one thread calls set/get while another one is calling set.

If we do have a multi-threaded possibility, recommend:
- Make the get/set method synchronized
- Add volatile keyword to indexPrefix